config/config.go

```diff
@@ -28,13 +33,16 @@ func Get() (*Configuration, error) {
 	}
 
 	cfg = &Configuration{
-		BindAddr: ":22300",
+		BindAddr:                   ":22300",
+		GracefulShutdownTimeout:    5 * time.Second,
+		HealthCheckInterval:        10 * time.Second,
```

Comment:
We're working on making all the health check intervals and timeouts consistent across the services. I didn't update the dp-recipe-api, because develop doesn't have the healthcheck yet. Can you update the config to the following values before merging this branch, please?
```
		HealthCheckInterval:               30 * time.Second,
		HealthCheckCriticalTimeout: 90 * time.Second,
```

Reply:
I have updated the healthcheck configs in PR https://github.com/ONSdigital/dp-recipe-api/pull/127